man-roxygen/bundle.R

```diff
@@ -0,0 +1,5 @@
+#' @section Saving prepped recipe object:
+#' 
+#' This recipe step may require native serialization methods to be properly 
+#' saved and/or passed between R sessions. To learn more about preparing 
+#' prepped recipes for serialization, see the bundle package.
```

Comment:
```suggestion
#' This recipe step may require native serialization when saving for 
#' use in another R session. To learn more about serialization for 
#' prepped recipes, see the [bundle](https://rstudio.github.io/bundle/) package.
```

Let's add a link, and hopefully make it more clear what might not work.